### PR TITLE
feat: navigation item from other locale copy [CU-2dzbdxg]

### PR DESCRIPTION
**Pull request by CodeVoyager**
## Ticket

https://app.clickup.com/t/2dzbdxg

depends on: https://github.com/VirtusLab-Open-Source/strapi-typed/pull/9

## Summary

- copying details of a navigation item from locale to locale
- minor ui performance tweaks

image

## Test Plan

- if locale is not selected in "Copy details from", "copy" button is inactive
- if locale is selected in "Copy details from", "copy" button is active
- if in source locale version there is matching node in navigation tree it will be applied to currently edited navigation item
- if found item's related content has localised version it will be used
- if found item's related content does not have localised version or content is not localised whatsoever, item will be applied "as-is"
- if matching node was not found a message will be displayed as select's error message
- if generic request error will happed a message will be displayed as select's error message
- generic and not found messages are different

### CODE REVIEW

**Comment by cyp3rius** [comment]
Task linked: [CU-2dzbdxg Copying navigation items between locales](https://app.clickup.com/t/2dzbdxg)

File: admin/src/pages/View/components/NavigationItemForm/index.js [hunk 1]
```diff
@@ -32,16 +37,29 @@ const NavigationItemForm = ({
   onCancel,
   getContentTypeEntities,
   usedContentTypesData,
-  appendLabelPublicationStatus = () => '',
+  appendLabelPublicationStatus = appendLabelPublicationStatusFallback, 
   locale,
+  readNavigationItemFromLocale,
 }) => {
+  const [isLoading, setIsLoading] = useState(isLoading);
```
Comment (ltsNotMike) [inline admin/src/pages/View/components/NavigationItemForm/index.js:44]:
```suggestion
  const [isLoading, setIsLoading] = useState(isPreloading);
```

_Uncaught ReferenceError: Cannot access 'isLoading' before initialization_

**Review by ltsNotMike — CHANGES_REQUESTED** [review]

File: admin/src/pages/View/components/NavigationItemForm/index.js (hunk 1 above)
Comment (CodeVoyager) [reply admin/src/pages/View/components/NavigationItemForm/index.js]: Updated

**Review by ltsNotMike — APPROVED** [review]

**Review by cyp3rius — APPROVED** [review]

**Comment by cyp3rius** [comment]
@CodeVoyager resolve conflict please and it's ready to be merged.

**Comment by CodeVoyager** [comment]
Issues are resolved @cyp3rius

**Pull request merged**